tests.php

```diff
@@ -0,0 +1,114 @@
+<?php
+	/* Tests script for phpcs-security-audit */
+
+	//eval($_GET['a']);
+	echo "aaaa";
+	echo "bbbb" . $_POST['b'];
+	echo "b";
+	db_query($_GET['a']);
+	preg_replace("/.*/ei", 'aaaaaaa', 'bbbbb');
+	preg_replace("/.*/ei", $_GET['a'], 'aaaaaaa');
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> warning - Usage of preg_replace with /e modifier is not recommended.